kuma/javascript/src/ga-provider.jsx

```diff
@@ -4,9 +4,13 @@ import { useContext, useEffect, useState } from 'react';
 
 export type GAFunction = (...any) => void;
 
-const noop: GAFunction = () => {};
+function ga(...args) {
+    if (typeof window === 'object' && typeof window.ga === 'function') {
```

Comment:
I'm wondering, if in the future we should consider making this "Ghostery"-proof like we do [here](https://github.com/mdn/kuma/blob/ea715ffdce74c9016854e74b5f28a25ef3f4d830/kuma/static/js/analytics.js#L40) in `kuma/static/js/analytics.js`? Something to think about, but not for this PR.